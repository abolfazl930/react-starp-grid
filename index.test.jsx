
import React from 'react';
import renderer from 'react-test-renderer';

import Col from './col';
import Row from './row';
import Container from './container';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Container className="test3">
        <Row className="test2">
          <Col className="test1" xs="3" md="5">
            Hello
          </Col>
        </Row>
        <Row>
          <Col>
            Hello
          </Col>
        </Row>
      </Container>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});