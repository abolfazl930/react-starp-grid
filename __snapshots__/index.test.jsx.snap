// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container test3"
>
  <div
    className=" row test2"
  >
    <div
      className="col-3 col-md-5 test1"
    >
      Hello
    </div>
  </div>
  <div
    className=" row "
  >
    <div
      className="col "
    >
      Hello
    </div>
  </div>
</div>
`;
